docs/common/COMPONENTS.md

```diff
@@ -239,6 +240,60 @@ const removeItem = handler<unknown, { items: Cell<Item[]>; item: OpaqueRef<Item>
 
 ---
 
+## ct-screen
+
+Full-screen container for app-like layouts. Use instead of `<div style={{ height: "100%" }}>` which doesn't work (parent has no explicit height).
+
+```tsx
+// ❌ DOESN'T WORK - content appears blank
+<div style={{ height: "100%", display: "flex", flexDirection: "column" }}>
+  {/* Content in DOM but invisible */}
+</div>
+
+// ✅ WORKS - full available width and height
+<ct-screen style="display: flex; flex-direction: column;">
+  <header>Title</header>
+  <div style="flex: 1; overflow: auto;">
+    {/* Scrollable content */}
+  </div>
+</ct-screen>
```

Comment:
The inline styles here are superfluous, `ct-screen` already styles `:host` with `display: flex; flex-direction: column;` internally so this is just noise.